types/types.d.ts

```diff
@@ -26,7 +26,12 @@ type FastifySession = FastifyPluginCallback<fastifySession.FastifySessionOptions
 }
 
 type Callback = (err?: Error) => void;
-type CallbackSession = (err: Error | null, result: Fastify.Session) => void;
+//type CallbackSession = (err: Error | null, result: Fastify.Session) => void;
+
+interface CallbackSession {
```

Comment:
I would have expected a type and not an interface:

type CallbackSession = ((err: null, result: Fastify.Session) => void) | ((err: Error) => void);

Reply:
Oh, this is cool. I didn't know you could do types like this. I switched it to a type, though, with a different syntax. Let me know if you would like me to change it to this type syntax.